### PR TITLE
Add hint to option-group options

### PR DESCRIPTION
**Pull request by andymoody**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,4 +27,5 @@ node_modules
 # Users Environment Variables
 .lock-wscript
 
+.idea
```
Comment (zuzak) [inline .gitignore]: Please don't put your personal IDE into .gitignore; you should [use a global configuration](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) instead.
Ideally .gitignore only contains things common to everybody using the repo.

Comment (andymoody) [reply .gitignore]: Cool, thanks, will remove!

**Review by zuzak — APPROVED** [review]

**Comment by zuzak** [comment]
(You might want to squash these commits)

**Comment by daniel-ac-martin** [comment]
> No description provided.

This PR doesn't / didn't contain any information on the purpose for this change. It also doesn't explain the reason to test against Node.js v7.

**Pull request merged**